card/lib/card/format.rb

```diff
@@ -504,10 +504,11 @@ module Location
       def page_path title, opts={}
         Rails.logger.warn "Pass only Card::Name to page_path #{title.class}, #{title}" unless Card::Name===title
         format = opts[:format] ? ".#{opts.delete(:format)}"  : ''
+        action = ( (path_opts = opts.delete(:path_opts)) && path_opts[:action] ) ? "#{opts[:path_opts][:action]}/" : ''
         query  = opts.present? ? "?#{opts.to_param}"         : ''
-        card_path "#{title.to_name.url_key}#{format}#{query}"
+        card_path "#{action}#{title.to_name.url_key}#{format}#{query}"
```

Comment:
ok, now I think I was getting confused between card_link (which uses "path") and card_path.  this is all pretty tangled still, I guess.